src/waypoint.jsx

```diff
@@ -201,7 +201,7 @@ export default class Waypoint extends React.Component {
       return Waypoint.below;
     }
 
-    if (contextScrollTop > waypointTop + thresholdPx) {
+    if (waypointTop + thresholdPx < contextScrollTop) {
```

Comment:
Could you also do that with line 200 (`if (waypointTop - thresholdPx > contextBottom)`)? See http://llewellynfalco.blogspot.com/2016/02/dont-use-greater-than-sign-in.html 😄